### ISSUE TITLE
App feels choppy while scrolling homepage

### ISSUE THREAD

**Issue by TilloX** [deckerst/aves#800]
**Describe the bug**
Main/Home screen feels choppy while scrolling. While rest of UI elements feels 120 Hz, homepage feels like 15 Hz while scrolling. 

**To Reproduce**
1. Open Aves Libre. 
2. Scroll vertically down/up to feel chopiness.

**Expected behavior**
It shouldn't feel choppy/ like 15 Hz and should behave like rest of the UI elements.

**Screenshots**
Sending screenrecord to mail.

**System information and logs:**
Package: deckers.thibault.aves.libre
Installer: com.looker.droidify
Aves version: 1.9.7-libre, build 10802
Flutter: stable 3.13.7
Android version: 12, API 31
Android build: S2RYAS32.58-13-12-5
Device: motorola moto g51 5G
Geocoder: ready
Mobile services: not available
System locales: en_IN
Storage volumes: /storage/emulated/0/, /storage/67FC-B19C/
Storage grants: 
Error reporting: false
[aves-logs-20231024_112544.txt](https://github.com/deckerst/aves/files/13111205/aves-logs-20231024_112544.txt)

**Comment by Niranjan-Dorage** [comment]
is this issue active ?
i would like to work ...

**Comment by Niranjan-Dorage** [comment]
@deckerst

**Comment by deckerst** [comment]
> is this issue active ? i would like to work ...

Not working on it anytime soon.

**Comment by Niranjan-Dorage** [comment]
its not open for work ? @deckerst

**Comment by deckerst** [comment]
[This repo does not accept PRs](https://github.com/deckerst/aves#code).

**Comment by M0-7** [comment]
Yeah I have the same issue the scrolling does not feel fluid in the app. I'm on a Google pixel 9

**Comment by deckerst** [comment]
Just tried with Aves v1.13.2 on a Sony Xperia 5 V at 120Hz and it's smooth.
@TilloX is it still an issue with the latest version of Aves?
